hack/verify-golint.sh

```diff
@@ -16,9 +16,10 @@
 
 set -euo pipefail
 
+export PATH=$(go env GOPATH)/bin:"${PATH}"
 if [[ -z "$(command -v golangci-lint)" ]]; then
   echo "Cannot find golangci-lint. Installing golangci-lint..."
-  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.31.0
```

Comment:
that's only for testing, is there still security issue here?

Reply:
Our vulnerability scanners don't distinguish for what the script is used for, only that it's in the repo. Even if it's in test it's still running in CI jobs etc.

Reply:
which vulnerability scanner are you referring?

Reply:
Internal google ones.

Reply:
that's hard way not using curl in testing sometimes, I think it's better exclude test folder if that's configurable?
actually this `hack/verify-golint.sh` could be deleted since there is already a github action doing same check:
https://github.com/kubernetes-csi/csi-driver-smb/blob/40bce9518749171801dd72ba7be94b4424ad0fb0/.github/workflows/static.yaml#L12-L16

Reply:
Deleting this sounds even better to me! I'll update the PR.